templates/ansible/verifiers/facts.yaml.erb

```diff
@@ -5,10 +5,19 @@
 -%>
 - name: verify that <%= object.name.underscore -%> was <%= lines(verbs[_state.to_sym]) -%>
   <%= module_name -%>:
+<% if object.facts.has_filters && object.facts.filter.name == 'filters' -%>
```

Comment:
what if it has filters but it isn't called 'filters'? is that ignored on purpose?

Reply:
There's two different cases:

1) GCE (the 'filters' notation)
2) Everything else (that I've seen so far)

I added some comments and a .is_gce? function to make this more clear.